lib/MemoryAnalysisModules/KillFlow.cpp

```diff
@@ -950,13 +950,13 @@ LoopAA::ModRefResult KillFlow::modref(const Instruction *i1,
 
       // check whether the pointer of the earlier op (from previous iter) is
       // killed before the load
-      else if (earlierPtr && pointerKilledBefore(L, earlierPtr, load,
-                                                 queryStart, AnalysisTimeout)) {
-        ++numKilledBackwardLoadFlows;
-        LLVM_DEBUG(errs() << "Removed the mod bit at AAA\n");
-        // res = ModRefResult(res & ~Mod);
-        res = NoModRef;
-      }
+      //else if (earlierPtr && pointerKilledBefore(L, earlierPtr, load,
```

Comment:
What was the commit associate with this?

Reply:
commit 8322cbac18653e72b062f294bcbc49e43349f6d1